content/blog/how-to-backup-and-restore-stateful-applications-on-kubernetes-using-kasten-k10-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,660 @@
+
+```shell
+$ kubectl get persistentvolumes 
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                                                                                                 STORAGECLASS               REASON   AGE
+
+pvc-3e55e9b3-097f-4ddf-bdcb-60825a7905ec   1Gi        RWO            Delete           Bound    mysql/mysql-pvc                                                                                                                           
+
+$ kubectl get persistentvolumeclaims -n mysql
+NAME        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS               AGE
+mysql-pvc   Bound    pvc-3e55e9b3-097f-4ddf-bdcb-60825a7905ec   1Gi        RWO            gl-sbp-frank-gl1-sstor01   9m50s
+```
+
+**2. Access MySQL database**
+
+In order to access MySQL database service using the mysql CLI, set first the port-forward of *service/mysql*:   
+
+```shell
+$ kubectl port-forward service/mysql -n mysql 42281:3306
+Forwarding from 127.0.0.1:42281 -> 3306
+Forwarding from [::1]:42281 -> 3306
+Handling connection for 42281
+```
+
+The d﻿eployed MySQL database service can be accessed by typing the following mysql command:
+
+```shell
+$ mysql -h 127.0.0.1 -uroot -pCfeDemo@123 -P 42281
+Welcome to the MariaDB monitor.  Commands end with ; or \g.
+Your MySQL connection id is 3
+Server version: 5.6.51 MySQL Community Server (GPL)
+
+Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+MySQL [(none)]> show databases;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| mysql              |
+| performance_schema |
++--------------------+
+3 rows in set (0,282 sec)
+```
+
+**3. Populate MySQL database** 
+
+The MySQL application repo has the *'test'* folder that contains a list of scripts for populating data records and testing the contents: 
+
+```shell
+$ tree mysql-app/test
+mysql-app/test
+├── employees.sql
+├── load_departments.dump
+├── load_dept_emp.dump
+├── load_dept_manager.dump
+├── load_employees.dump
+├── load_salaries1.dump
+├── load_salaries2.dump
+├── load_salaries3.dump
+├── load_titles.dump
+├── show_elapsed.sql
+├── test_employees_md5.sql
+└── test_employees_sha.sql
+```
+
+Typing the following command to populate a sample *employees* data to the MySQL database:
+
+```shell
+$ cd mysql-app/test
+$ mysql -h 127.0.0.1 -uroot -pCfeDemo@123 -P 42281 < employees.sql
+INFO
+CREATING DATABASE STRUCTURE
+INFO
+storage engine: InnoDB
+INFO
+LOADING departments
+INFO
+LOADING employees
+INFO
+LOADING dept_emp
+INFO
+LOADING dept_manager
+INFO
+LOADING titles
+INFO
+LOADING salaries
+data_load_time_diff
+NULL
+```
+
+T﻿he added sample data records *employees* can be checked and verified by running below commands:
```

Comment:
Modify to "The added sample data records **called** employees can be checked and verified by **running the commands shown below:"**